src/building/suggested.md

```diff
@@ -3,6 +3,19 @@
 The full bootstrapping process takes quite a while. Here are some suggestions
 to make your life easier.
 
+## Installing a `pre-commit` hook
```

Comment:
Actually, one more nit: I don't think it's necessary to have "pre-commit" as inline code.

```suggestion
## Installing a pre-commit hook
```